src/app/greencity/modules/places/places.component.spec.ts

```
@@ -134,99 +140,221 @@ describe('PlacesComponent', () => {
+      zoom: 8,
+      panTo: jasmine.createSpy('panTo'),
+      setCenter: jasmine.createSpy('setCenter')
+    } as any;
+
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  afterEach(() => {
+    delete window.google;
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)